main.tf

```diff
@@ -5,7 +5,7 @@ resource "aws_guardduty_detector" "eu_west_1" {
 
 resource "aws_guardduty_detector" "eu_west_2" {
   enable   = true
-  provider = "aws"
+  provider = "aws.eu-west-2"
```

Comment:
I did this on purpose, it doesn't seem to like having only providers with aliases